src/components/pages/HomePage/index.jsx

```diff
@@ -36,16 +30,20 @@ import { ReactComponent as DeployImg } from "../../../assets/content/svg/deploy.
 import { ReactComponent as ShopImg } from "../../../assets/content/svg/shop.svg";
 import { ReactComponent as VaultImg } from "../../../assets/content/svg/vault.svg";
 import { ReactComponent as PartyImg } from "../../../assets/content/svg/party.svg";
+//#endregion
 
-let backgroundPictures = [baristaBg, breadBg, shopBg];
+//#region > Functions
+const backgroundPictures = [baristaBg, breadBg, shopBg];
 
 function getRandomInt() {
   return Math.floor(Math.random() * Math.floor(backgroundPictures.length));
 }
 
 // Get randomized image
-let randomBG = backgroundPictures[getRandomInt()];
+const randomBG = backgroundPictures[getRandomInt()];
+//#endregion
```

Comment:
Constants should not be included in the function block. Constant variables that are initialized should also be written completely in capital letters.

Reply:
Microsoft -1 again.. GitHub does not show your changes :). But I saw somehow that you changed the `Functions` to `Initialisers`or something.. but this isn't init stuff. You should implement two separate sections:
- Functions
- Constant Variabels

Even when `getRandomInt()`is used once at the moment it might be used more often in the future.. Therefore -> Own sections. Thx

```
Functions:
 function getRandomInt() {
  return Math.floor(Math.random() * Math.floor(backgroundPictures.length));
}

Constant Variables:
const backgroundPictures = [baristaBg, breadBg, shopBg];
// Get randomized image
let randomBG = backgroundPictures[getRandomInt()];
const randomBG = backgroundPictures[getRandomInt()];

```

Reply:
Made seperate sections. Thanks for the heads up!